src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -77,7 +77,7 @@ public function testFixManagersAutoMappingsWithTwoAutomappings()
             'SecondBundle' => 'My\SecondBundle',
         ];
 
-        $reflection = new \ReflectionClass(\get_class($this->extension));
+        $reflection = new \ReflectionClass($this->extension::class);
```

Comment:
actually `ReflectionClass` constructor supports objects as well

Reply:
Right! Let use the object directly everywhere possible.

Reply:
Good catch , thanks